api/Navigator.json

```diff
@@ -1448,10 +1448,10 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": "71"
+              "version_added": false
```

Comment:
This is `navigator.mediaSession`, and in my testing Firefox 81 on macOS it's not there. It is in Firefox Developer Edition, so probably behind some flag. But the existing data is wrong, glad to fix some false positives too!